torch/special/__init__.py

```diff
@@ -27,3 +27,31 @@
     tensor([ 0.5724,  0.0000, -0.1208])
 
 """)
+
+entr = _add_docstr(_special.special_entr,
+                   r"""
+entr(input, *, out=None) -> Tensor
+
+Computes the entropy on :attr:`input`, elementwise.
```

Comment:
"entropy" is a vague term and we should use the opportunity to elaborate.

Would it be more accurate to say this computes the components of Shannon's entropy function? (See https://en.wikipedia.org/wiki/Entropy_(information_theory).)

It might also be helpful to reminder the reader that the computation is in base 2, so this operation produces "bits" or "shannons."